src/main/scala/dotty/dokka/tasty/BasicSupport.scala

```diff
@@ -11,50 +12,42 @@ trait BasicSupport:
   extension SymbolOps on (sym: reflect.Symbol):
     def packageName(using ctx: Context): String = 
       if (sym.isPackageDef) sym.fullName 
-      else sym.owner.packageName
+      else sym.maybeOwner.packageName
     
     def topLevelEntryName(using ctx: Context): Option[String] = if (sym.isPackageDef) None else
       if (sym.owner.isPackageDef) Some(sym.name) else sym.owner.topLevelEntryName
 
-    def dri = asDRI(sym)
-    def declarationDri = asDRI(sym, true)   
+    // TODO make sure that DRIs are unique plus probably reuse semantic db code?  
+    def dri =
+      if sym == Symbol.noSymbol then emptyDRI else
+        val pointsTo = 
+          if (!sym.isTypeDef) PointingToDeclaration.INSTANCE 
+          else PointingToGenericParameters(sym.owner.typeMembers.indexOf(sym))
```

Comment:
I'm not certain what the intention is in the `sym.owner.typeMembers.indexOf(sym)` part. Are we trying to find the index of a generic parameter of a class? If so, this code looks like it works because of implementation details and may not work for classes with type members. The correct way to find that index would probably be by calling `.typeParams` on the class symbol.